rel/common/nodetool

```diff
@@ -76,6 +79,30 @@ main(Args) ->
                 Other ->
                     io:format("~p\n", [Other])
             end;
+        ["ready"] ->
+            case rpc:call(TargetNode, application, which_applications, [], 60000) of
+                {badrpc, Reason} ->
+                    io:format("RPC to ~p failed: ~p\n", [TargetNode, Reason]),
+                    halt(1);
+                List ->
+                    case lists:foldl(fun(_, ok) ->
+                                            ok;
+                                       ({leo_gateway, _, _}, _) ->
+                                            ok;
+                                       ({leo_manager, _, _}, _) ->
+                                            ok;
+                                       ({leo_storage, _, _}, _) ->
+                                            ok;
+                                       (_, Acc) ->
+                                            Acc
+                                    end, waiting, List) of
+                        ok ->
+                            io:format("Application is ready\n");
+                        _ ->
+                            io:format("Application is not ready\n"),
+                            halt(1)
+                    end
```

Comment:
http://erlang.org/doc/man/lists.html#any-2 would be more suitable one as it stops to iterate when Predictor func returns true.

```erlang
case lists:any(
fun({leo_gateway,_,_}) ->
    true;
fun({leo_manager,_,_}) ->
    true;
fun({leo_storage,_,_}) ->
    true;
fun(_) ->
    false
end, List) of ->
    true ->
    false ->
end